public/resources/index.ejs

```diff
@@ -0,0 +1,19 @@
+<p class="text-body">Would you like to be listed in this page? Fill out this <a href="https://docs.google.com/a/rangle.io/forms/d/1qzWaDpTgTPe4iPDRF_VCT9aHXKimUocwlFnVJUdKabY/viewform?c=0&w=1">form</a>.</p>
+
+<div class="js-resources-ctrl resources" ng-controller="ResourcesCtrl as ctrl">
+    <div class="section" ng-repeat="(category, categoryContent) in ctrl.fbObject">
+        <div>
+            <h1 class="capitalize">{{category}}</h1>
+            <div ng-repeat="section in categoryContent">
+                <div class="grid-fluid" ng-repeat="(subsectionName, subsectionArray) in section">
+                    <div>
+                        <h2 class="capitalize">{{ subsectionName }}</h2>
+                        <ul ng-repeat="resource in subsectionArray.resources">
+                            <li ng-if="resource.rev"><a target="_blank" href="{{resource.url}}">{{ resource.title }}</a></li>
```

Comment:
@juleskremer I am making an explicit check for approved entries of the resource page.